native/src/modules/app/components/Navigator.js

```diff
@@ -65,9 +65,7 @@ type PropsType = {|
   fetchCategory: (cityCode: string, language: string, key: string, forceUpdate: boolean) => void,
   fetchCities: (forceRefresh: boolean) => void,
   routeKey: ?string,
-  routeName: ?string,
-  cityCode: ?string,
-  languageCode: ?string
+  routeName: ?string
```

Comment:
Why have you removed city and language? Is this part of the ticket and needed or refactoring?

Reply:
Refactoring, was not used anywhere anymore.